tests/nginx.conf

```diff
@@ -0,0 +1,23 @@
+worker_processes 1;
+error_log stderr;
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+    access_log  /var/log/nginx/access.log  main;
+
+    # make non-root can write here
+    client_body_temp_path /tmp/nginx_client_temp 1 2;
+    fastcgi_temp_path /tmp/nginx_fastcgi_temp 1 2;
+    proxy_temp_path /tmp/nginx_proxy_temp 1 2;
+    scgi_temp_path /tmp/nginx_scgi_temp 1 2;
+    uwsgi_temp_path /tmp/nginx_uwsgi_temp 1 2;
```

Comment:
A temporary and ugly fix before https://github.com/macbre/docker-nginx-http3/issues/92 is implemented.